packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [35.1.1]
+
+### Fixed
```

Comment:
Maybe also worth mentioning the TypeScript updates? I know we typically omit `devDependency` changes, but we make an exception for cases like this that do have an impact upon the published code/types.

```suggestion
### Changed

- Upgrade TypeScript version to `~5.2.2` ([#4576](https://github.com/MetaMask/core/pull/4576), [#4584](https://github.com/MetaMask/core/pull/4584))

### Fixed
```